modules/sdk-coin-sui/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@bitgo/sdk-coin-sui",
+  "version": "1.0.0",
+  "description": "BitGo SDK coin library for Mysten",
```

Comment:
```suggestion
  "description": "BitGo SDK coin library for Sui",
```

Reply:
The coin is Sui correct, not the company Mysten?

Reply:
Good catch! updated all with Sui